### PR TITLE
enh(epp): hitachi nas snmp proc v2

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md [hunk 1]
```diff
@@ -1,35 +1,356 @@
...
+* HW-Storage-Hitachi-Hnas-SNMP-custom
+
+Il apporte les modèles de service suivants :
+
+| Alias                  | Modèle de service                                   | Description                                              | Défaut | Découverte |
+|:-----------------------|:----------------------------------------------------|:---------------------------------------------------------|:-------|:-----------|
+| Cluster-Status         | HW-Storage-Hitachi-Hnas-Cluster-Status-SNMP         | Contrôle l'état des noeuds du cluster                    |        |            |
+| Hardware-Global        | HW-Storage-Hitachi-Hnas-Hardware-Global-SNMP        | Contrôle le matériel                                     | X      |            |
+| Interfaces             | HW-Storage-Hitachi-Hnas-Interfaces-SNMP             | Contrôle les interfaces                                  |        | X          |
+| Virtual-Volumes-Quotas | HW-Storage-Hitachi-Hnas-Virtual-Volumes-Quotas-SNMP | Contrôle l'utilisation des quotas des volumes virtuelles |        |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md:23]:
```suggestion
| Virtual-Volumes-Quotas | HW-Storage-Hitachi-Hnas-Virtual-Volumes-Quotas-SNMP | Contrôle l'utilisation des quotas des volumes virtuels |        |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md [hunk 2]
```diff
@@ -1,35 +1,356 @@
...
+| Volume-Usage-Global    | HW-Storage-Hitachi-Hnas-Volume-Usage-Global-SNMP    | Contrôle l'utilisation des volume                        | X      |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md:24]:
```suggestion
| Volume-Usage-Global    | HW-Storage-Hitachi-Hnas-Volume-Usage-Global-SNMP    | Contrôle l'utilisation des volumes                        | X      |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md [hunk 3]
```diff
@@ -1,35 +1,356 @@
...
+
+### Règles de découverte
+
+| Nom de la règle                             | Description                                               |
+|:--------------------------------------------|:----------------------------------------------------------|
+| HW-Storage-Hitachi-Hnas-SNMP-Interface-Name | Discover the disk partitions and monitor space occupation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md:30]: description en anglais

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md [hunk 4]
```diff
@@ -1,35 +1,356 @@
...
+| *volume_name*#volume.space.usage.bytes | B     |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+### Configuration SNMP
+
+Afin de superviser votre **Hitachi NAS** en SNMP, il est nécessaire de configurer l'agent sur le serveur comme indiqué sur la documentation officielle.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md:94]: lien?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md [hunk 5]
```diff
@@ -1,35 +1,356 @@
...
+
+## Installation
+
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet
+n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
+
+Au contraire, si la plateforme utilise une licence *offline*, installez le paquet
+sur le **serveur central** via la commande correspondant au gestionnaire de paquet
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md:109]:
```suggestion
sur le **serveur central** via la commande correspondant au gestionnaire de paquets
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md [hunk 6]
```diff
@@ -1,35 +1,359 @@
...
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-### Centreon Plugin
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your **Hitachi NAS**
+server.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md:95]: link?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md (hunk 4 above)
Comment (garnier-quentin) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md:94]: we don't have a link

File: pp/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md (hunk 6 above)
Comment (garnier-quentin) [reply pp/integrations/plugin-packs/procedures/hardware-storage-hitachi-hnas-snmp.md:95]: we don't have a link

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**